openlibrary/templates/account/readinglog_stats.html

```diff
@@ -13,7 +13,7 @@
 $jsdef render_works_list(works):
     <ul class="works-list">
     $for work in works:
-        <li><a href="$work.key" style="font-style:oblique">$work.title</a>
+        <li><a href="$work.key" style="font-style:oblique">$work.title $work.subtitle</a>
```

Comment:
There should be a `: ` between the title and sub title. Make sure not display the `:` if there is no subtitle!